scripts/select_appointments.sql

```diff
@@ -1 +1,6 @@
 -- retrieve appointment information
+SELECT start, patient_id  FROM appointments
+WHERE doctor_id=7
+
+SELECT start, doctor_id FROM appointments
+WHERE patient_id=9
```

Comment:
Besides the issue with non-existent `start`, `patient_id`, and `doctor_id` columns, using the id of a specific object is never the right answer. You only know what that number is because you have direct access to the data.   Also, you are assuming that that id number will never change. The only way to get Dr. Hubbard's appointments is to query for them. 

```sql
SELECT p.surname, a.date, a.time
  FROM patients p
  INNER JOIN appointments a ON a.patient_id = p.id
  INNER JOIN doctors d ON a.doctor_id = d.id
    WHERE d.surname = 'Hubbard';

SELECT d.surname, a.date, a.time
  FROM doctors d
  INNER JOIN appointments a ON a.doctor_id = d.id
  INNER JOIN patients p ON a.patient_id = p.id
    WHERE p.surname = 'Wilcox';
```